modules/sor/sorV2/lib/poolsV3/stable/stablePool.ts

```diff
@@ -103,6 +103,7 @@ export class StablePool implements BasePoolV3 {
             pool.dynamicData.tokenPairsData as TokenPairData[],
             pool.liquidityManagement,
             hookState,
+            // @ts-ignore: Ignore TypeScript error for accessing name property
             pool.hook?.name,
```

Comment:
wouldn't it be cleaner to pass the name via the hookState? easier to manage if everything is within one scope.

Reply:
addressed the change request with the latest pushes.